extensions/amp-story/1.0/amp-story.css

```diff
@@ -377,6 +377,10 @@ amp-story-grid-layer {
   left: 0 !important;
 }
 
+amp-story-page amp-video video {
```

Comment:
You can have multiple videos within a story page:
1. Publishers content within a grid-layer
2. Videos in an Ad, that can have any layout
3. Videos in an amp-story-page-attachment, that is non full-bleed scrollable content

In this case, we want our selector to be specific enough so it only targets the first one.
As Jon mentioned, publishers still need to be able to have non full-bleed videos, so our selector needs to reflect that.

We actually already have such a selector here, but because we want to make the `position: absolute;` non overridable, we need to copy/paste it into `amp-story.css`:

https://github.com/ampproject/amphtml/blob/f134a84168e00b5dec63deb5bb70e9799e3cd4dc/extensions/amp-story/1.0/amp-story-user-overridable.css#L30-L35

Reply:
Do you think I actually want to cover these img tags too, or just use ` .i-amphtml-story-grid-template-fill amp-video video` ? 
Do they have the same problem with a base-64 loading image that breaks without position absolute?

Reply:
Yes they do, good point! Let's add `.i-amphtml-story-grid-template-fill amp-img img` too